app/build.gradle

```diff
@@ -67,14 +66,15 @@ android {
 dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.5.0'
-    implementation 'androidx.appcompat:appcompat:1.3.0'
-    implementation 'com.google.android.material:material:1.3.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
-    implementation 'androidx.lifecycle:lifecycle-livedata-ktx:2.3.1'
-    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1'
+    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'androidx.appcompat:appcompat:1.4.1'
+    implementation 'com.google.android.material:material:1.5.0'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.3'
+    implementation "androidx.lifecycle:lifecycle-livedata-ktx:$version_androidx_lifecycle"
+    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:$version_androidx_lifecycle"
 
+//    localImplementation project(path: ':q42stats')
```

Comment:
Why? This breaks the buildvariants, I think.

Perhaps the buildvariants are confusing and should be removed. I made them to debug the publishing process where I could select through build variants wether the app would use Q42Stats from local source or from jitpack. 
But now that that's fixed, the buildVariants might better be removed to avoid confusion. What  do you think

Reply:
Ah, I hardly looked at this. I'll set the local as default, that will solve it. Having the flavors is fine as long as I don't have to think about it.